t/mojo/collection.t

```diff
@@ -5,167 +5,186 @@ use Mojo::ByteStream qw(b);
-
-# reduce
-$collection = c(2, 5, 4, 1);
-is $collection->reduce(sub { $a + $b }), 12, 'right result';
-is $collection->reduce(sub { $a + $b }, 5), 17, 'right result';
-is c()->reduce(sub { $a + $b }), undef, 'no result';
-
-# sort
-$collection = c(2, 5, 4, 1);
-is_deeply $collection->sort->to_array, [1, 2, 4, 5], 'right order';
-is_deeply $collection->sort(sub { $b cmp $a })->to_array, [5, 4, 2, 1], 'right order';
-is_deeply $collection->sort(sub { $_[1] cmp $_[0] })->to_array, [5, 4, 2, 1], 'right order';
-$collection = c(qw(Test perl Mojo));
-is_deeply $collection->sort(sub { uc(shift) cmp uc(shift) })->to_array, [qw(Mojo perl Test)], 'right order';
-$collection = c();
-is_deeply $collection->sort->to_array, [], 'no elements';
-is_deeply $collection->sort(sub { $a cmp $b })->to_array, [], 'no elements';
-
-# uniq
-$collection = c(1, 2, 3, 2, 3, 4, 5, 4);
-is_deeply $collection->uniq->to_array, [1, 2, 3, 4, 5], 'right result';
-is_deeply $collection->uniq->reverse->uniq->to_array, [5, 4, 3, 2, 1], 'right result';
-$collection = c([1, 2, 3], [3, 2, 1], [3, 1, 2]);
-is_deeply $collection->uniq(sub { $_->[1] }), [[1, 2, 3], [3, 1, 2]], 'right result';
-$collection = c(c(1, 2), c(1, 2), c(2, 1));
-is_deeply $collection->uniq(join => ',')->flatten->to_array, [1, 2, 2, 1], 'right result';
-$collection = c(undef, '', 3, 2, 1, 0);
-is_deeply $collection->uniq->to_array, [undef, 3, 2, 1, 0], 'right result';
-$collection = c(undef, '', 3, 2, 1, 0);
-is_deeply $collection->uniq(sub {$_})->to_array, [undef, 3, 2, 1, 0], 'right result';
-
-# TO_JSON
-is encode_json(c(1, 2, 3)), '[1,2,3]', 'right result';
-
-# head
-$collection = c(1, 2, 5, 4, 3);
-is_deeply $collection->head(0)->to_array, [], 'right result';
-is_deeply $collection->head(1)->to_array, [1], 'right result';
-is_deeply $collection->head(2)->to_array, [1, 2], 'right result';
-is_deeply $collection->head(-1)->to_array, [1, 2, 5, 4], 'right result';
-is_deeply $collection->head(-3)->to_array, [1, 2], 'right result';
-is_deeply $collection->head(5)->to_array, [1, 2, 5, 4, 3], 'right result';
-is_deeply $collection->head(6)->to_array, [1, 2, 5, 4, 3], 'right result';
-is_deeply $collection->head(-5)->to_array, [], 'right result';
-is_deeply $collection->head(-6)->to_array, [], 'right result';
-
-# tail
-$collection = c(1, 2, 5, 4, 3);
-is_deeply $collection->tail(0)->to_array, [], 'right result';
-is_deeply $collection->tail(1)->to_array, [3], 'right result';
-is_deeply $collection->tail(2)->to_array, [4, 3], 'right result';
-is_deeply $collection->tail(-1)->to_array, [2, 5, 4, 3], 'right result';
-is_deeply $collection->tail(-3)->to_array, [4, 3], 'right result';
-is_deeply $collection->tail(5)->to_array, [1, 2, 5, 4, 3], 'right result';
-is_deeply $collection->tail(6)->to_array, [1, 2, 5, 4, 3], 'right result';
-is_deeply $collection->tail(-5)->to_array, [], 'right result';
-is_deeply $collection->tail(-6)->to_array, [], 'right result';
+subtest 'Array' => sub {
```

Comment:
I see that there is a capital `A` here, whilst the other subtests (except the one right below this) use all-lowercase. Is this intentional or should this be converted to lowecase as well for consistency?

Reply:
They use lowercase because they are method names. This is not a method name.

Reply:
Ah, right. It was not clear to me that that was a convention that was followed.